chain/pool/txpool.go

```diff
@@ -175,6 +176,34 @@ func (tp *TxnPool) processTx(txn *Transaction) error {
 			}
 
 		}
+	case pb.NanoPaymentChannelType:
```

Comment:
txlist sorts transactions by nonce. This NanoPaymentChannelType transaction has no nonce.  Insert transactions with this type will cause an error.

Reply:
every tx has a nonce, because it's on the same level as payload and not optional. it could be 0 or any other value though, since it's not checked, so the sort order would be wrong. what error this will cause?

Reply:
fixed